src/content/docs/how-to-work-on-coding-challenges.mdx

```diff
@@ -99,16 +106,16 @@ Extra information for a challenge, in markdown
 
 # --seed--
 
-## --before-user-code--
+## --before-user-code-- (deprecated)
 
-```lang
-Code evaluated before the user’s code.
```

Comment:
the linter was protesting for this, so I put it with `js`, which is what we usually use anyway